include/lsst/jointcal/Jointcal.h

```diff
@@ -20,9 +20,9 @@ namespace jointcal {
     struct JointcalControl {
         LSST_CONTROL_FIELD(sourceFluxField, std::string, "name of flux field in source catalog");
 
-        JointcalControl(std::string const sourceFluxType) :
+        JointcalControl(std::string const _sourceFluxField="slot_CalibFlux") :
```

Comment:
I suggest passing by reference (`const &` instead of `const`). I also suggest using the same variable name as the attribute (no leading underscore); I admit that can look a bit confusing, but it works a leading underscore should be reserved to indicate "private" (or protected)